backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/PredicateHandlerUtils.java

```diff
@@ -49,7 +49,7 @@ protected String applyOperator(
     }
   }
 
-  protected String wrapValue(OperatorEnum.Operator operator, String value) {
+  protected Object wrapValue(OperatorEnum.Operator operator, Object value) {
```

Comment:
I really, really don't like this. Can we do something better with concrete types?

Reply:
return type is based on condition so it is not possible to set return type with concrete type. i am sorry. otherwise argument i have fixed with generic argument